base/binaryplatforms.jl

```diff
@@ -144,7 +144,7 @@ end
 
 # Hash definitino to ensure that it's stable
 function Base.hash(p::Platform)
```

Comment:
I don't know the use-case here, but shouldn't this be `Base.hash(::Platform, ::UInt)` as recommended by the `hash` docs?

Reply:
Thanks!  I clearly didn't read the docs properly.  ;)